benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -40,6 +40,14 @@ msgstr ""
 "Obtenga una tarifa reducida en los autobuses del Santa Barbara MTD cuando "
 "toque para viajar"
 
+#, fuzzy
+#| msgid "Get a reduced fare on SacRT buses when you tap to ride"
+msgid ""
+"Get a reduced fare on Ventura County Connects buses when you tap to ride"
+msgstr ""
+"Obtenga una tarifa reducida en los autobuses del SacRT cuando toque para "
+"viajar"
```

Comment:
Just need to update this one!

Reply:
Thanks for this catch! I rebased on #3005 to avoid migrations conflicts, and addressed this in the new [b03319f](https://github.com/cal-itp/benefits/pull/3017/commits/b03319f4c2b14ae09d23c34bcf9284371a5d0ebd)